examples/benzeneselenol.cjson

```diff
@@ -0,0 +1,279 @@
+{
+  "chemical json": 0,
+  "name": "Benzeneselenol",
+  "inchi": "1S/C6H6Se/c7-6-4-2-1-3-5-6/h1-5,7H",
+  "formula": "C 6 H 6 Se",
+  "atoms": {
+    "elements": {
+      "number": [  6,  6,  6,  6,  6,  6,  1,  1,  1,  1,  1, 34,  1 ]
+    },
+    "coords": {
+      "3d": [  -2.894762	-0.013642	 -0.001528
```

Comment:
I think he said they were handwritten, and I guess it was a mistake in copying them? I had missed this, good catch. Several other arrays seem to suffer from the same issue below.